samples/test/util.py

```diff
@@ -263,6 +263,7 @@ class KfpTask:
     '''A KFP runtime task'''
     name: str
     type: str
+    state: int
```

Comment:
for discussion: can we convert the state to a str corresponding to the enum like "COMPLETE", "CACHED"? this would help making test runtime diff readable.

Reply:
If we want to convert the state to a str, it means that in our e2e test, we need to maintain the pairing between int and string in e2e test which is an implementation detail in ml_metadata. I think the code itself is readable even though the test runtime diff is not. Devs are still able to figure out when they take a look at the test code.

Reply:
makes sense